plugins/org.jkiss.dbeaver.ui.editors.sql/src/org/jkiss/dbeaver/ui/editors/sql/SQLEditorPropertyTester.java

```diff
@@ -125,13 +128,20 @@ public boolean test(Object receiver, String property, Object[] args, Object expe
                 return editor.isFoldingEnabled();
             case PROP_FOLDING_SUPPORTED:
                 return editor.getProjectionAnnotationModel() != null;
+            case PROP_OVERLAY_FOCUSED: {
```

Comment:
Why do you need a dedicated property for this? Can you perform this check in the "can execute" branch?